cmd/tools/studio/cfg/copyenv.go

```diff
@@ -0,0 +1,85 @@
+package cfg
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+var heyCMD = &cobra.Command{
+	Use:   "copyenv",
+	Short: "Copy env variables configmap for a microservice in one environment to another environment",
+	Long: `
+	Copy env variables configmap for a microservice in one environment to another environment.
+
+	go run main.go tools studio cfg copyenv --application cde2e951-d40a-3548-8b45-64c0ded97940 \
+		--microservice-name frontend \
+		--from-env test \
+		--to-env prod
+	`,
+	Run: func(cmd *cobra.Command, args []string) {
+		logrus.SetFormatter(&logrus.JSONFormatter{})
+		logrus.SetOutput(os.Stdout)
+		logContext := logrus.StandardLogger()
+
+		sourceMicroserviceName := viper.GetString("tools.studio.cfg.microservice-name")
+		sourceEnvironment := viper.GetString("tools.studio.cfg.from-env")
+		destinationEnvironment := viper.GetString("tools.studio.cfg.to-env")
+		application := viper.GetString("tools.studio.cfg.application")
+
+		sourceConfigMapName := fmt.Sprintf("%s-%s-env-variables", sourceEnvironment, sourceMicroserviceName)
+		namespace := fmt.Sprintf("application-%s", application)
+		logContext.WithFields(logrus.Fields{
+			"k8sNamespace":     namespace,
+			"k8sConfigmap":     sourceConfigMapName,
+			"microserviceName": sourceMicroserviceName,
+		}).Info("Begins to copy env var configmap")
+
+		kubeCtlGetCfgMap := exec.Command("kubectl", "-n", namespace, "get", "configmap", sourceConfigMapName, "-o", "json")
+		_, err := kubeCtlGetCfgMap.StderrPipe()
+		if err != nil {
+			logContext.Fatal(err)
+			return
+		}
+
+		configMapJson, err := kubeCtlGetCfgMap.Output()
+		if err != nil {
+			logContext.WithFields(logrus.Fields{
+				"k8sNamespace": namespace,
+				"k8sConfigMap": sourceConfigMapName,
+				"error":        err,
+			}).Fatal("Failed to get configmap")
+			return
+		}
```

Comment:
```suggestion
		ctx := context.TODO()
		k8sClient, _ := platformK8s.InitKubernetesClient()
		configMap, err := k8sClient.CoreV1().ConfigMaps(namespace).Get(ctx, sourceConfigMapName, metav1.GetOptions{})
		if err != nil {
			logContext.Fatal(err)
		}
```

Suggestion code not exact and missing imports, but should give you the idea.

Instead of just using raw `exec` powers, I suggest to use the k8s go SDK instead. If this command stays inside the `studio` subcommand then it already has the flags setup for kubeconfig, then you only need to call initialize the k8s client in the command and start using it.
https://github.com/dolittle/platform-api/blob/f643513e96152fdaf0675542258cbfc914114c88/cmd/tools/studio/root.go#L35-L41

Reply:
@joelhoisko  I think this is a good idea, but I don't think I want to spend more time on it. I think we can write this off as tech debt. When we incorporate copy of env variables into the platform-api we can rewrite it. Make sense?